bddl/activity_definitions/cleaning-up-branches-and-twigs/problem0.bddl

```diff
@@ -1,11 +1,11 @@
-(define (problem cleaning-up-branches-and-twigs_0)
```

Comment:
Let's actually do branch.n.02 in this task. I think stalk is a bit too high-level. My portion of the BDDLs already assume stalk.n.02 is not a leaf.